pkg/api/frontend_logging_test.go

```diff
@@ -137,8 +142,8 @@ func indexOf(arr []interface{}, item string) int {
 	return -1
 }
 
-func soContextContains(logRecord *log.Record, label string, value interface{}) {
-	So(logRecord.Ctx, ShouldContain, label)
+func assertContextContains(t *testing.T, logRecord *log.Record, label string, value interface{}) {
+	assert.Contains(t, logRecord.Ctx, label)
 	labelIdx := indexOf(logRecord.Ctx, label)
-	So(logRecord.Ctx[labelIdx+1], ShouldEqual, value)
+	assert.Equal(t, logRecord.Ctx[labelIdx+1], value)
```

Comment:
With `assert.Equal` the expectation comes before the actual value.
```suggestion
	assert.Equal(t, value, logRecord.Ctx[labelIdx+1])
```